src/App.tsx

```diff
@@ -1,11 +1,78 @@
-import { useState } from 'react';
+import React, { useState, useMemo, useEffect } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
+import { MovieData } from './types/MovieData';
+// import { ResponseError } from './types/ReponseError';
+
+import { getMovie } from './api';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [query, setQuery] = useState('');
+  const [submitedQuery, setSubmitedQuery] = useState('');
+  const [findedMovie, setFindedMovie] = useState<MovieData | null>(null);
+  const [movieNotFounded, setMovieNotFounded] = useState(false);
```

Comment:
Consider renaming the variable 'movieNotFounded' to 'movieNotFound' for better naming convention